server/src/fetchers/relationship-fetchers.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import type { Viewer } from '../session/viewer';
+import {
+  type RelationshipErrors,
+  undirectedStatus,
+  directedStatus,
+} from 'lib/types/relationship-types';
+
+import _groupBy from 'lodash/fp/groupBy';
+import invariant from 'invariant';
+
+import { dbQuery, SQL } from '../database';
+
+type RelationshipOperation =
+  | 'delete_directed'
+  | 'friend'
+  | 'pending_friend'
+  | 'know_of';
+type UserRelationshipOperations = {
+  [string]: $ReadOnlyArray<RelationshipOperation>,
+};
+
+async function fetchFriendRequestRelationshipOperations(
+  viewer: Viewer,
+  userIDs: string[],
+) {
+  const query = SQL`
+    SELECT user1, user2, status
+    FROM relationships_directed
+    WHERE
+      (user1 IN (${userIDs}) AND user2 = ${viewer.userID}) OR
```

Comment:
Let's return this query to what it was before. I think I may have confused you with my previous comments. I don't think there was any need to change this